### PR TITLE
fix(vital-cards): update cards with buttons & misc fixes

### PR DESCRIPTION
**Pull request by hvanyo**

### CODE REVIEW

File: sites/vital-demo/src/data/pages/styleguide/buttons/demo-link-arrow.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "href": "https://www.cnn.com/"
```
Comment (dewen) [inline sites/vital-demo/src/data/pages/styleguide/buttons/demo-link-arrow.json:2]: shall we use generic demo link like 'example.com' ?

File: sites/vital-demo/src/data/pages/styleguide/buttons/demo-link$Link1.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "text": "a"
```
Comment (dewen) [inline sites/vital-demo/src/data/pages/styleguide/buttons/demo-link$Link1.json:2]: "a" => "download"

File: sites/vital-demo/src/data/pages/styleguide/buttons/demo-link-arrow$Link1.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "text": "asf"
```
Comment (dewen) [inline sites/vital-demo/src/data/pages/styleguide/buttons/demo-link-arrow$Link1.json:2]: "asf" => "example.com"

File: packages/vital-card/src/components/Card/tokens/Hero.ts [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+ * limitations under the License.
+ */
+
+import omit from 'lodash/omit';
+import {
+  flowHoc, replaceWith, Div, extendMeta, H1, H4, as
+} from '@bodiless/fclasses';
+// import { asBodilessLink } from '@bodiless/components-ui';
+import { vitalImage } from '@bodiless/vital-image';
+// import { vitalLink } from '@bodiless/vital-link';
```
Comment (dewen) [inline packages/vital-card/src/components/Card/tokens/Hero.ts:21]: remove commented line

File: packages/vital-card/src/components/Card/tokens/Hero.ts [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import omit from 'lodash/omit';
+import {
+  flowHoc, replaceWith, Div, extendMeta, H1, H4, as
+} from '@bodiless/fclasses';
+// import { asBodilessLink } from '@bodiless/components-ui';
```
Comment (dewen) [inline packages/vital-card/src/components/Card/tokens/Hero.ts:19]: remove commented line

File: sites/vital-demo/src/data/pages/styleguide/buttons/demo-link-arrow$Link1.json (hunk 3 above)
Comment (hvanyo) [reply sites/vital-demo/src/data/pages/styleguide/buttons/demo-link-arrow$Link1.json:2]: Resolved

File: sites/vital-demo/src/data/pages/styleguide/buttons/demo-link-arrow.json (hunk 1 above)
Comment (hvanyo) [reply sites/vital-demo/src/data/pages/styleguide/buttons/demo-link-arrow.json:2]: resolved

**Review by dewen — APPROVED** [review]

**Review by konst8 — APPROVED** [review]

**Pull request merged**